### PR TITLE
Refactor/remove box

### PR DESCRIPTION
**Pull request by Brooooooklyn**
A rough fix for https://github.com/gtk-rs/cairo/issues/183

***Note***
This PR introduced some breaking changes

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -49,8 +49,10 @@ pub use error::{
 };
 
 pub use patterns::{
-    //Traits
+    // Enums
     Pattern,
+    //Traits
```
Comment (GuillaumeGomez) [inline src/lib.rs]: Please add a whitespace after `//`.

**Comment by GuillaumeGomez** [comment]
Just one nit but everything else seems good, thanks!

**Comment by EPashkin** [comment]
@GuillaumeGomez Travis passed here, `cargo test` and building examples works for me on Windows with latest cairo

**Comment by GuillaumeGomez** [comment]
That's weird... I'm not too happy to merge if the CI is going to fail on every new PR to cairo but if it works for you, then it's fine for me. Can you still take a look at what's going wrong please?

**Comment by EPashkin** [comment]
@GuillaumeGomez see https://github.com/gtk-rs/cairo/issues/185 problem seems "self-solved", just need wait until apveyor update mingw64 on image.
Or we add update in your script for cairo.

**Comment by GuillaumeGomez** [comment]
Oh cool! And as you prefer.

**Comment by Brooooooklyn** [comment]
@GuillaumeGomez would you please update Cargo version? So I could update my project

**Pull request merged**